README.md

```diff
@@ -48,3 +50,29 @@ AUTH_AUTO_ERROR: bool = True
 AUTH_ALLOW_NONE: bool = False
 
 ```
+
+Пример мока библиотеки:
+
+Установите нужные завивсимости
+```shell
+pip install 'auth-lib-profcomff[testing]'
+```
+
+```python
+import pytest
+from fastapi.testclient import TestClient
+from fastapi import FastAPI
+
+@pytest.fixture
+def client(auth_mock):
+    yield TestClient(FastAPI())
+
+@pytest.mark.authenticated("scope1", "scope2", ...)
```

Comment:
```
  /Users/dyakovri/Desktop/test/tests/test_one.py:9: PytestUnknownMarkWarning: Unknown pytest.mark.authenticated - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.authenticated("scope1", "scope2")
```